### PR TITLE
Table styles

### PR DESCRIPTION
**Pull request by ikyle53**
- Fixed the table header's from tr's to th's in the tables. 
- Added reference text to table cells where JS will anchor to it. 
- Colored individual cells so they're visible.

Feel free to change/undo styles as needed.

### CODE REVIEW

**Comment by ikyle53** [comment]
> * Fixed the table header's from 's to 's in the tables.
> * Added reference text to table cells where JS will anchor to it.
> * Colored individual cells so they're visible.
> 
> Feel free to change/undo styles as needed.

File: build.html [hunk 1]
```diff
@@ -14,9 +14,9 @@
 <header>
   <nav>
     <ul>
-      <li><a href="#">Home</a></li>
+      <li><a href="index.html">Home</a></li>
```
Comment (nojronatron) [inline build.html:17]: Please make sure the nav links work before submitting code. I fixed it on 3 files and will approve this PR.

File: css/style.css [hunk 2]
```diff
@@ -26,3 +26,18 @@ div:hover {
 p {
   text-align: center;
 }
+
+
+/*******************  table styles  *******************/
+
```
Comment (nojronatron) [inline css/style.css:32]: Nice! This is an excellent placeholder until we get the cart implemented in JS.

**Review by nojronatron — APPROVED** [review]
CSS on table for visibility is great thansk!
Please be sure to keep nav links working in future commits.

File: build.html (hunk 1 above)
Comment (ikyle53) [reply build.html:17]: Thanks for the catch!

**Review by egbeale — APPROVED** [review]

**Review by Dsmul — APPROVED** [review]

**Pull request merged**